fdbcli/MetaclusterCommands.actor.cpp

```diff
@@ -418,14 +494,15 @@ std::vector<const char*> metaclusterHintGenerator(std::vector<StringRef> const&
 
 CommandFactory metaclusterRegisterFactory(
     "metacluster",
-    CommandHelp("metacluster <create_experimental|decommission|register|remove|configure|list|get|status> [ARGS]",
-                "view and manage a metacluster",
-                "`create_experimental' and `decommission' set up or deconfigure a metacluster.\n"
-                "`register' and `remove' add and remove data clusters from the metacluster.\n"
-                "`configure' updates the configuration of a data cluster.\n"
-                "`list' prints a list of data clusters in the metacluster.\n"
-                "`get' prints the metadata for a particular data cluster.\n"
-                "`status' prints metacluster metadata.\n"),
+    CommandHelp(
+        "metacluster <create_experimental|decommission|register|remove|restore|configure|list|get|status> [ARGS]",
+        "view and manage a metacluster",
+        "`create_experimental' and `decommission' set up or deconfigure a metacluster.\n"
+        "`register' and `remove' add and remove data clusters from the metacluster.\n"
+        "`configure' updates the configuration of a data cluster.\n"
```

Comment:
Can you also add a line with a brief description of what `restore` does to the blurb here?